sips/sip-031/sip-031.md

```diff
@@ -0,0 +1,575 @@
+# Preamble
+
+**SIP Number:** 031
+
+**Title:** SIP-031: Five-Year Stacks Growth Emissions
+
+**Author(s):**
+- Mitchell Cuevas (Stacks Foundation)
+- Alex Miller (Hiro)
+- Tycho Onnasch (Zest)
+- Jakob Schillinger (Hermetica)
+- Andre Serrano (Bitcoin L2 Labs)
+- Dylan Floyd (Bitflow)
+
+**Type:** Economic, Governance
+
+**Status:** Draft
+
+**Created:** 6/12/2025
+
+**License:** To be determined
+
+**Sign-off:** Pending
+
+**Discussions:** 
+- [[DRAFT] Fueling Stacks Builders & Growth, Meet SIP-031](https://forum.stacks.org/t/draft-fueling-stacks-builders-growth-meet-sip-031/18060/)
+- [SIP 31.1 - A New Draft Arrives](https://forum.stacks.org/t/sip-31-1-a-new-draft-arrives/18130 )
+- [Workshops: SIP-031 Appointments Committee](https://forum.stacks.org/t/workshops-sip-031-appointments-committee/18141/8 )
+  
+
+---
+
+# Abstract
+
+* Stacks’ goal as an ecosystem is to empower the creation of the Bitcoin economy \- we envision that all projects that want to build on Bitcoin, from DeFi to gaming, memecoins to NFTs, and marketplaces to identity, should be built on top of Stacks.  
+* Stacks is transitioning from a tech-first to a traction-first phase, fueled by a new endowment and streamlined operations.  
+* The proposal raises fresh capital to put Stacks on par with other top projects, addressing historic underfunding and unlocking the ability to compete at scale.  
+* A new operational entity will scale execution, while a community-governed endowment ensures transparent, accountable, and sustainable growth.  
+* This is Stacks’ decisive step to accelerate adoption, empower builders of all sizes, and drive long-term value for all stakeholders.  
```

Comment:
Wordsmitthing -- it's not possible to know if this step is "decisive" since determining this can only be done in hindsight.  It's certainly a significant step though.